### DISCUSSION TITLE
对于动态大小的节点

### DISCUSSION THREAD

**Discussion by zmcode** [antvis/X6#2463] [Q&A]
有什么办法可以动态设置它的大小, 例如我的节点需要使用使用图片和文字, 图片的大小都不一样, 文字也会不一样, 我现在必须先创建一个dom, 塞到body中, 计算出实际大小, 在生成这个节点, 这让就会多一个生成dom的操作了

**Comment by NewByVector** [comment]
这个没有办法，需要事先知道节点大小或者计算出内容大小后进行大小调整。
